go-controller/pkg/node/obj_retry_gateway.go

```diff
@@ -188,3 +189,7 @@ func (h *gwEventHandler) RecordDeleteEvent(obj interface{}) {}
 func (h *gwEventHandler) RecordSuccessEvent(obj interface{}) {}
 
 func (h *gwEventHandler) RecordErrorEvent(obj interface{}, reason string, err error) {}
+
+func (h *gwEventHandler) RecordMaxRetryFailedMetric() {
```

Comment:
just call the metric directly and increment imo.

Reply:
sure, done.